quay-enterprise/bitbucket-app.md

```diff
@@ -0,0 +1,19 @@
+# Creating an OAuth Application in BitBucket
+
+- Log into BitBucket
+- Visit the Settings page for your organization
+- Click the "OAuth" tab under "Access Management"
+- Click the Add Consumer button:
+
+<img src="img/bb-add-consumer.png" class="image-center"/>
+
+- Enter your registry's URL as the `URL`
```

Comment:
"Next, configure BitBucket to redirect you back to Quay upon a successful login:"